src/ui/Controllers/CourseController.cs

```diff
@@ -79,17 +79,22 @@ public async Task<IActionResult> ShowPublish(string instCode, string accrediting
             if (result)
             {
                 TempData["MessageType"] = "success";
-                TempData["MessageTitle"] = "Your course has been published";
-                var searchUrl = searchAndCompareUrlService.GetCoursePageUri(course.InstCode, course.CourseCode);
+
                 if (featureFlags.ShowCourseLiveView)
                 {
+                    TempData["MessageTitle"] = "Your course has been published and will be updated within 1 working day";
```

Comment:
How about:
"Your course has been published. Your changes will be on the website within 1 working day, usually within 2 hours."

`The link for this course is:<br/>[full url]`